src/views/HomeRedesign/CDKTypeTabs.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  Flex,
+  forwardRef,
+  Heading,
+  Tab,
+  TabList,
+  TabPanelProps,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Text,
+} from "@chakra-ui/react";
+import type { FunctionComponent } from "react";
+import { useHistory } from "react-router-dom";
+import { CDKType, CDKTYPE_RENDER_MAP } from "../../constants/constructs";
+import { useCatalogResults } from "../../hooks/useCatalogResults";
+import { getSearchPath } from "../../util/url";
+import { SECTION_PADDING } from "./constants";
+import { PackageGrid } from "./PackageGrid";
+
+interface PackageTabProps {
+  cdkType?: CDKType;
+  data: ReturnType<typeof useCatalogResults>;
+}
+
+const PackageTab: FunctionComponent<PackageTabProps> = ({ cdkType, data }) => {
+  return (
+    <Tab isDisabled={data.page.length < 1}>
+      {cdkType ? CDKTYPE_RENDER_MAP[cdkType].name : "All CDKs"} (
+      {data.results.length})
+    </Tab>
+  );
+};
+
+const PackageTabPanel = forwardRef<PackageTabProps & TabPanelProps, "div">(
+  ({ cdkType, data, ...props }, ref) => {
+    const { push } = useHistory();
+
+    const onSeeAllClick = () => {
+      window.scrollTo(0, 0);
+      push(getSearchPath({ cdkType }));
+    };
+
+    return (
+      <TabPanel ref={ref} {...props} p={0}>
+        <PackageGrid packages={data.page} />
+
+        <Flex justify="center" w="full">
+          <Button colorScheme="blue" my={8} onClick={onSeeAllClick}>
+            See all {cdkType ? CDKTYPE_RENDER_MAP[cdkType].name + " " : ""}
+            constructs
+          </Button>
+        </Flex>
+      </TabPanel>
+    );
+  }
+);
+
+export const CDKTypeTabs: FunctionComponent = () => {
+  const anyCDKType = useCatalogResults({ limit: 4 });
+  const awscdk = useCatalogResults({ cdkType: CDKType.awscdk, limit: 4 });
+  const cdk8s = useCatalogResults({ cdkType: CDKType.cdk8s, limit: 4 });
+  const cdktf = useCatalogResults({ cdkType: CDKType.cdktf, limit: 4 });
```

Comment:
nit: Make the sorting mechanism explicit (i.e. by downloads, relevance, etc.)

Reply:
I believe we want these sorted by downloads, so the only real blocker is that we don't have sort by downloads merged yet!